### PR TITLE
Add new rules

### PR DESCRIPTION
**Pull request by cordear**
Add two rules for F95Zone.

### CODE REVIEW

File: dist/mydata.json [hunk 1]
```diff
@@ -3589,5 +3589,33 @@
...
+    "nextLink": "//div[@class='pages']//li[b]/following-sibling::li[1]/a",
+    "category": ["forum", "nsfw"]
+  },
+  {
+    "name": "south-plus-thread",
+    "url": "^https?://www.south-plus.org/thread",
+    "pageElement": "//*[@id='ajaxtable']/tbody[2]",
+    "nextLink": "//div[@class='pages']//li[b]/following-sibling::li[1]/a",
+    "category": ["forum", "nsfw"]
   }
```
Comment (machsix) [inline dist/mydata.json:3620]: 我已在https://github.com/machsix/Super-preloader/commit/5d769b074e34429371eeb2bd1ca90afb39015490 中修改了原有的规则，请删掉这些行

File: dist/mydata.json [hunk 2]
```diff
@@ -3589,5 +3589,33 @@
     "exampleUrl": "https://www.cool18.com/bbs4/index.php?app=forum&act=cachepage&cp=tree1",
     "nextLink": "//*[@id='d_list_page']/a[contains(text(),'下一页')]",
     "category": ["forum", "nsfw"]
+  },
+  {
+    "name": "F95zone Others",
+    "url": "^https?:\/\/f95zone.to\/forums\/(?!general)",
```
Comment (machsix) [inline dist/mydata.json:3595]: `/`不需要转义，`.`需要转义

File: dist/mydata.json [hunk 3]
```diff
@@ -3589,5 +3589,33 @@
...
+    "pageElement": "//*[@id='top']/div[4]/div/div[1]/div/div/div/div[4]/div[2]/div[2]",
```
Comment (machsix) [inline dist/mydata.json:3596]: 请尝试使用class name、id、或所包含的文字定义规则，这种依据index的容易出问题

**Comment by cordear** [comment]
I have changed some rules to follow the code style.
- delete duplicated rules about south-plus.
- modify rules about F95Zone with more reliable Xpath.

**Pull request merged**